tutorials/tutorial1.ipynb

```diff
@@ -164,862 +175,1989 @@
-    "           --cleanpeakbw dsc.Mono.2400.cutsites.smoothed.200.3.narrowPeak.bw \\\n",
-    "           --intervals val_intervals.bed \\\n",
-    "           --out_dir ./ \\\n",
-    "           --prefix Mono.50.2400.val \\\n",
-    "           --pad 5000"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "This produces a .h5 file in the current directory (`Mono.50.2400.val.h5`) containing the validation data for the model.\n",
+    "We provide a chromosome sizes file 'hg19.auto.sizes' that contains sizes for all the autosomes of the hg19 reference genome. We split off chromosome 2 to use as the validation set, and chromosome 10 to use as the holdout set, and use the remaining autosomes as the training set. Since a whole chromosome is too long to feed into the model at once, we split each of these chromosomes into 50,000-bp long intervals.\n",
     "\n",
+    "This command trains a deep learning model using the supplied clean and noisy ATAC-seq data, for 10 epochs (10 full passes through the dataset). At the end of every epoch, the current state of the model is saved in the directory `atacworks_train_latest`, and the performance of the current model is measured on the validation set. At the end, out of the 10 saved models, the one with the best performance on the validation set is saved as `atacworks_train_latest/model_best.pth.tar`\n",
     "\n",
-    "## Step 7: Train and validate a model using the parameters in the given config files\n",
+    "This model has learned a mapping from the 50-cell signal to the 2400-cell signal and peak calls. Given a new 50-cell ATAC-seq track, it can denoise the track and produce high-quality peak calls.\n",
     "\n",
-    "We next train an AtacWorks model to learn a mapping from the noisy (50-cell) ATAC-seq signal to the clean (2400-cell) ATAC-seq signal and peak calls. The two .yaml files that we downloaded into the `configs` directory contain all the parameters that describe how to train the model. `configs/model_structure.yaml` contains parameters that control the architecture of the model and  `configs/config_params.yaml` contains parameters that control the process of training, such as the learning rate and batch size.\n",
+    "See [Tutorial 2](tutorial2.ipynb) for step-by-step instructions on how to apply this trained model to another dataset.\n",
     "\n",
-    "To train the model, we supply the training and validation datasets as well as the two config files."
+    "To change any of the parameters for the deep learning model, copy paste the default config files at `$atacworks/configs` to the current location. Edit the appropriate parameters in `./configs/train_config.yaml` or `./configs/model_structure.yaml`. You can provide the custom config file paths to atacworks using `--config` and `--config_mparams` flags. Look at Appendix 1 for the example command. Type `atacworks train --help` for an explanation of the parameters.\n"
```

Comment:
This is very unclear. I think we need to create a new appendix where we show steps to copy the default config files to the working directory and how to modify them and supply modified config files in the training command.

Reply:
changed.

Reply:
I think this needs to be a separate appendix. Have left a comment to that effect.

Reply:
done.